### ISSUE TITLE
formBuilder's icons conflict with other icon fonts

### ISSUE THREAD

**Issue by andreasotto** [kevinchappell/formBuilder#918]
### Description:

I'm already using another icon font in my project in the css class space `icon-*`.
Since installing formBuilder, the icons are broken, because formBuilder overrides that css class space with its own icons. How can i customize the namespace formBuilder uses?

### Environment Details:

 - formBuilder Version: 3.1.3
 - Browser: doesnt matter
 - OS: doesnt matter

### Expected Behavior

fb should have a configurable option to give its css namespace for icons its own name?

### Actual Behavior

Broken icons in the global css space

### Steps to Reproduce

example: use formBuilder together with Font IcoMoon which uses `icon-*` css classes.

**Comment by andreasotto** [comment]
Not a nice solution but a workaround for the moment is as follows:

First in formBuilder the `font-family: fb-icons` has the be removed for the css classes `[class*=" icon-"]:before,[class^=icon-]:before` in the code.

Than you can enable the `font-family: fb-icons` again beeing more specific: 

```
.form-wrap.form-builder .frmb-control li {
    font-family: inherit;
    display: block;
}

.form-wrap.form-builder [class^=icon-]:before, .form-wrap.form-builder [class*=" icon-"]:before {
    font-family: fb-icons;
}
```
The result is that only `icon-*` classes use the `fb-icons` font family when they are with the formBuilder's form wrapper.

**Comment by johndeen** [comment]
@andreasotto I am facing same issue. Please elaborate what changes are required and where.
I have created a pen online please check it out:
**[https://codepen.io/johndeen/pen/QXVNRw](https://codepen.io/johndeen/pen/QXVNRw)**
I am using icomoon in my project, when i add form builder script it removes fonts from my project.
Found that font face in the script file caused the issue. kindly help me out.

**Comment by burgoyne** [comment]
I am also running into this issue, but with font awesome instead of icomoon. An official fix would be appreciated, as this one doesn't seem to work for me.

**Comment by johndeen** [comment]
Sorry i wasn't able to understand but i managed to solve it:
1) Created fontello files and changed the prefix.
2) Used embedded css file from fontello and replaced css in the formbuilder.min.js.
3) updated prefex in the js file accordingly.
Attaching beautified version with changes.
[formBuilder-FontIssue-TempFix.zip](https://github.com/kevinchappell/formBuilder/files/3363459/formBuilder-FontIssue-TempFix.zip)

**Comment by suryacaprice** [comment]
Facing the same issue when it will be fixed in the repo

**Comment by suryacaprice** [comment]
Can someone raise a pull request to resolve this

**Comment by ppshein** [comment]
I've fixed how to make it happen by adding following code snippet into project. That's to define `formbuilder-icons` font will be used in specific area.

```
  .form-wrap.form-builder .frmb-control li::before {
    font-family: "formbuilder-icons", fb-icons;
  }

  .form-wrap.form-builder [class^=icon-]:before, .form-wrap.form-builder [class*=" icon-"]:before {
    font-family: "formbuilder-icons", fb-icons;
  }
```

For the rest of whole project, we could be able to define to use original font like that

```
  .nav-icon [class^=nav-icon]:before, .nav [class*=" nav-icon"]:before {
    font-family: "simple-line-icons";
  }
```

**Comment by lucasnetau** [comment]
Conflict was resolved in 3.5.2 (August 24th 2020) https://github.com/kevinchappell/formBuilder/releases/tag/v3.5.2

Fixed by https://github.com/kevinchappell/formBuilder/commit/1231aae178edad3dad8cdffb753020db3bc6a24c
